src/Jackett.Common/Indexers/TorrentSyndikat.cs

```diff
@@ -173,35 +173,38 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
 
             try
             {
-                CQ dom = results.Content;
-                var rows = dom["table.torrent_table > tbody > tr"];
-                var globalFreeleech = dom.Find("legend:contains(\"Freeleech\")+ul > li > b:contains(\"Freeleech\")").Any();
+                var parser = new HtmlParser();
+                var dom = parser.ParseDocument(results.Content);
+                var rows = dom.QuerySelectorAll("table.torrent_table > tbody > tr");
+                var globalFreeleech = dom.QuerySelector("legend:contains(\"Freeleech\")+ul > li > b:contains(\"Freeleech\")") != null;
                 foreach (var row in rows.Skip(1))
                 {
                     var release = new ReleaseInfo();
                     release.MinimumRatio = 1;
                     release.MinimumSeedTime = 96 * 60 * 60;
 
-                    var qRow = row.Cq();
-
-                    var catStr = row.ChildElements.ElementAt(0).FirstElementChild.GetAttribute("href").Split('=')[1].Split('&')[0];
+                    var catStr = row.Children[0].FirstElementChild.GetAttribute("href").Split('=')[1].Split('&')[0];
                     release.Category = MapTrackerCatToNewznab(catStr);
 
-                    var qLink = row.ChildElements.ElementAt(2).FirstElementChild.Cq();
-                    release.Link = new Uri(SiteLink + qLink.Attr("href"));
-                    var torrentId = qLink.Attr("href").Split('=').Last();
-
-                    var descCol = row.ChildElements.ElementAt(1);
-                    var qCommentLink = descCol.FirstElementChild.Cq();
-                    var torrentTag = descCol.Cq().Find("span.torrent-tag");
-                    var torrentTags = torrentTag.Elements.Select(x => x.InnerHTML).ToList();
-                    release.Title = qCommentLink.Attr("title");
-                    release.Description = string.Join(", ", torrentTags);
-                    release.Comments = new Uri(SiteLink + qCommentLink.Attr("href").Replace("&hit=1", ""));
+                    var qLink = row.Children[2].FirstElementChild;
+                    release.Link = new Uri(SiteLink + qLink.GetAttribute("href"));
+                    var torrentId = qLink.GetAttribute("href").Split('=').Last();
+
+                    var descCol = row.Children[1];
+                    var torrentTag = descCol.QuerySelectorAll("span.torrent-tag");
+                    if (torrentTag.Any())
+                    {
+                        var torrentTags = torrentTag.Select(x => x.InnerHtml).ToList();
+                        release.Description = string.Join(", ", torrentTags);
```

Comment:
Remove `.ToList()` for performance. [`string.Join()` can accept IEnumerable](https://docs.microsoft.com/en-us/dotnet/api/system.string.join?view=netframework-4.8#System_String_Join_System_String_System_Collections_Generic_IEnumerable_System_String__), so no need to allocate a temporary list.
```suggestion
                        release.Description = string.Join(", ", torrentTag.Select(x => x.InnerHtml));
```